src/alire/alire-publish.adb

```diff
@@ -522,6 +539,80 @@ package body Alire.Publish is
       end if;
    end Show_And_Confirm;
 
+   -------------------
+   -- Verify_Github --
+   -------------------
+
+   procedure Verify_Github (Context : in out Data) is
+      pragma Unreferenced (Context);
+   begin
+
+      --  Early return if forcing
+
+      if Force then
+         Trace.Warning ("GitHub checks skipped");
+         return;
+      end if;
+
+      --  User has an account
+
+      if not Config.Defined (Config.Keys.User_Github_Login) then
```

Comment:
You can use `Query_Config.User_GitHub_Login` here it will ask the user for it's login or take it from the config.

Reply:
Yes, I use it later, I make this check to print a bit of extra context before asking for the first time.

Reply:
I see for this one. The other one above I don't understand.